transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -33,8 +33,8 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    static {
-        // Ensure JNI is initialized by the time this class is loaded by this time!
+    {
```

Comment:
@normanmaurer yep, allows you to avoid duplicating code when you have multiple constructors and comes in handy with anonymous classes (where you cannot use a constructor).